fabfile/adage_server.py

```diff
@@ -210,6 +210,7 @@ def init_instance():
 def build_interface():
     """ have grunt perform a deployment build for us """
     with cd(CONFIG['interface_dir']), \
+            shell_env(PHANTOMJS_BIN='/usr/local/bin/phantomjs'), \
```

Comment:
The unit tests that are run by Karma does need phantomjs, but **karma-phantomjs-launcher** is installed locally in `node_modules` sub-directory, not in `/usr/local/bin/`.